tasklist/els-schema/src/main/java/io/camunda/tasklist/schema/SchemaStartup.java

```diff
@@ -38,33 +41,42 @@ public class SchemaStartup {
 
   @PostConstruct
   public void initializeSchema() throws MigrationException {
-    LOGGER.info("SchemaStartup started.");
-    LOGGER.info("SchemaStartup: validate schema.");
-    schemaValidator.validate();
-
-    Boolean isCreateSchema = false;
-    if (TasklistProperties.ELASTIC_SEARCH.equalsIgnoreCase(tasklistProperties.getDatabase())) {
-      isCreateSchema = tasklistProperties.getElasticsearch().isCreateSchema();
-    } else if (TasklistProperties.OPEN_SEARCH.equalsIgnoreCase(tasklistProperties.getDatabase())) {
-      isCreateSchema = tasklistProperties.getOpenSearch().isCreateSchema();
-    }
+    try {
+      LOGGER.info("SchemaStartup started.");
+      LOGGER.info("SchemaStartup: validate index versions.");
+      schemaValidator.validateIndexVersions();
+      LOGGER.info("SchemaStartup: validate index mappings.");
+      final Map<IndexDescriptor, Set<IndexMappingProperty>> newFields =
+          schemaValidator.validateIndexMappings();
+      final boolean createSchema =
+          tasklistProperties.getDatabaseType().equals("opensearch")
```

Comment:
`TasklistProperties.OPEN_SEARCH.equalsIgnoreCase(tasklistProperties.getDatabase())` as we do in other places